feasibility-triangle/troubleshooting/test-cql.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
```

Comment:
there is no Readme on how to use the script in the repository neither is there an explanation in the script itself.
Either change the existing troubleshooting.md or add a help message.
Note, that someone troubleshooting should be made aware of the get-fhir-server-access-token.sh in the repository.